CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python-contrib/compare/v0.19b0...HEAD)
 
 ### Changed
+- Downgrade DataDog exporter's `ddtrace` dependency so the exporter continues to work.
```

Comment:
Would it be more accurate to say that the version range has been restricted? We've never prescribed a specific version.

Reply:
Yes. Updated.